app/models/batch_entry.rb

```diff
@@ -1,4 +1,5 @@
 class BatchEntry < ActiveRecord::Base
+  serialize :result, Array
```

Comment:
Can we serialize without the Array bit?  BatchJob and BatchEntry itself is very generic, so making it Array forces it into a certain constraint that isn't really required.